android/app/src/main/kotlin/app/revanced/manager/flutter/MainActivity.kt

```diff
@@ -384,13 +384,7 @@ class MainActivity : FlutterActivity() {
 
                 updateProgress(0.9, "Signing...", "Signing APK")
 
-                try {
-                    Signer("ReVanced", keystorePassword)
-                        .signApk(patchedFile, outFile, keyStoreFile)
-                } catch (e: Exception) {
-                    print("Error signing APK: ${e.message}")
-                    e.printStackTrace()
-                }
+                Signer("ReVanced", keystorePassword).signApk(patchedFile, outFile, keyStoreFile)
```

Comment:
Why not instead simply return after the stack trace is printed? Also when would you expect an error here?

Reply:
> Why not instead simply return after the stack trace is printed?

But I think, if an exception is occurred, the exception should be handled at this catch block:

https://github.com/ReVanced/revanced-manager/blob/a9878dbbdfb25cde985fc49a81a8e862eb51ec8c/android/app/src/main/kotlin/app/revanced/manager/flutter/MainActivity.kt#L396-L405

Looking at other parts of the method, I don't see any exception handling code.

> Also when would you expect an error here?

If user entered an incorrect password

<details>

<summary>Screenshot</summary>

![Screenshot_20231205-164905](https://github.com/ReVanced/revanced-manager/assets/90122968/07838c6d-4e2f-4454-affe-4b3966092c8a)

</details>

If user imported a keystore generated by Android Studio

<details>

<summary>Screenshot</summary>

![Screenshot_20231205-182526](https://github.com/ReVanced/revanced-manager/assets/90122968/810fb1b8-cc22-4a67-97ab-1579d57f270d)

</details>

(These screenshots are from this PR code)

Reply:
Currently the stack trace is printed using `print()`, but this only outputs to stdout and does not appear on the screen.
So user can't see the error.
This problem can be seen in the screen recording at #1253.